diff -ur Mozilla-CA-20221114.orig/lib/Mozilla/CA.pm Mozilla-CA-20221114/lib/Mozilla/CA.pm
--- Mozilla-CA-20221114.orig/lib/Mozilla/CA.pm	2022-11-14 02:43:44.000000000 -0800
+++ Mozilla-CA-20221114/lib/Mozilla/CA.pm	2023-05-06 09:40:17.166348913 -0700
@@ -8,8 +8,13 @@
 use File::Basename qw(dirname);
 
 sub SSL_ca_file {
+    my $curlcert = File::Spec->catfile(File::Spec->rootdir(), "etc", "pki", "tls", "certs", "ca-bundle.crt");
     my $file = File::Spec->catfile(dirname(__FILE__), "CA", "cacert.pem");
-    return File::Spec->rel2abs($file);
+    if (-s $curlcert) {
+        return ($curlcert);
+    } else {
+        return File::Spec->rel2abs($file);
+    }
 }
 
 1;
